content/en/docs/topics/registries.md

```diff
@@ -244,19 +275,4 @@ The following example contains a
 Migrating from classic [chart repositories]({{< ref "chart_repository.md" >}})
 (index.yaml-based repos) is as simple using `helm pull`, then using `helm push` to upload the resulting `.tgz` files to a registry.
 
-## Deprecated features and strict naming policies
```

Comment:
Since you're removing this anchor, you need to edit the item above and remove this link to it:

> currently a strict requirement (\[more info here](#deprecated-features-and-strict-naming-policies)).

This is why the build is currently broken in this PR.

Reply:
thank you @bridgetkromhout 

removed link